frontend/src/pages/SendMetadata/MetadataForm.jsx

```diff
@@ -0,0 +1,205 @@
+        method: 'PUT',
+        body: JSON.stringify(state),
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      });
+      // assuming that any successful response is a JSON object
+      const { status, ok } = res;
+      if (!ok) {
+        const err = new Error();
+        err.status = status;
+        throw err;
+      }
+      setSubmissionStatus('success');
+    } catch (err) {
+      dispatch(alertActions.error('Failed to post dataset'));
+    }
+  };
+
+  if (submissionStatus === 'success') {
+    return <Redirect to="/viewData" />;
+  }
+
+  const {
+    metadataTypeName, releaseState, description, formatName, municipalityName, url,
+  } = state;
+
+  return (
+    <Wrapper>
+      <StyledForm>
+        <Select name="metadataTypeName" value={metadataTypeName} onChange={handleChange}>
+          <option value="">Metadata type</option>
+          {metadataTypes.map(({ name }) => <option key={name} value={name}>{name}</option>)}
+        </Select>
+        <p>
+          Are none of these types appropriate?
+          {' '}
+          <Link to="/newMetadataType">
+            <b>Create a new one</b>
+          </Link>
+        </p>
+        <HorizontalWrapper>
+          <RadioLabel htmlFor="bluelight" background="#9999dd" border="#6666aa">
+            <Input type="radio" name="releaseState" value={1} id="bluelight" checked={releaseState === 1} onChange={handleRadioChange} />
+            {' Released'}
+          </RadioLabel>
+          <RadioLabel htmlFor="greenlight" background="#ccffcc" border="#00ff00">
+            <Input type="radio" name="releaseState" value={2} id="greenlight" checked={releaseState === 2} onChange={handleRadioChange} />
+            {' Ready for release'}
+          </RadioLabel>
+          <RadioLabel htmlFor="yellowlight" background="#ffffcc" border="#ffff00">
+            <Input type="radio" name="releaseState" value={3} id="yellowlight" checked={releaseState === 3} onChange={handleRadioChange} />
+            {' Needs work'}
+          </RadioLabel>
+          <RadioLabel htmlFor="redlight" background="#ffcccc" border="#ff5555">
+            <Input type="radio" name="releaseState" value={4} id="redlight" checked={releaseState === 4} onChange={handleRadioChange} />
+            {' Unreleasable'}
+          </RadioLabel>
+        </HorizontalWrapper>
+        <TextArea placeholder="description" name="description" value={description} onChange={handleChange} />
+        <Select name="formatName" value={formatName} onChange={handleChange}>
+          <option value="">Data format</option>
+          {dataFormats.map((format) => <option key={format} value={format}>{format}</option>)}
+        </Select>
+        <Select name="municipalityName" value={municipalityName} onChange={handleChange}>
+          <option value="">Municipality</option>
+          {
+            municipalities.map(({ name }) => (<option key={name} value={name}>{name}</option>))
```

Comment:
This should only display the municipalities that the user is allowed to add a dataset to. For now the backend only support one municipality, the users main municipality, its named 'municipalityName' in the user object.